src/platform/forms-system/src/js/web-component-patterns/internationalPhonePattern.jsx

```diff
@@ -0,0 +1,50 @@
+import PhoneNumberReviewWidget from 'platform/forms-system/src/js/review/PhoneNumberWidget';
+import VaTextInputField from 'platform/forms-system/src/js/web-component-fields/VaTextInputField';
+
+/**
+ * Web component v3 uiSchema for international phone number
+ *
+ * ```js
+ * examplePhone: internationalPhoneUI() // Phone number
+ * examplePhone: internationalPhoneUI('Cell phone number')
+ * examplePhone: internationalPhoneUI({
+ *   title: 'Cell phone number',
+ *   hint: 'This is a hint'
+ * })
+ * examplePhone: {
+ *  ...internationalPhoneUI('Main phone number')
+ * }
+ * ```
+ * @param {string | UIOptions & {
+ *   title?: UISchemaOptions['ui:title'],
+ *   hint?: string,
+ * }} [options] accepts a single string for title, or an object of options
+ * @returns {UISchemaOptions}
+ */
+export const internationalPhoneUI = options => {
+  const { title, ...uiOptions } =
+    typeof options === 'object' ? options : { title: options };
+
+  return {
+    'ui:title': title ?? 'Phone number',
+    'ui:webComponentField': VaTextInputField,
+    'ui:reviewWidget': PhoneNumberReviewWidget,
+    'ui:autocomplete': 'tel',
+    'ui:options': {
+      inputType: 'tel',
+      ...uiOptions,
+    },
+    'ui:errorMessages': {
+      required: 'Please enter a phone number (with or without dashes)',
+      pattern: 'Please enter a valid phone number (with or without dashes)',
+    },
```

Comment:
Phone error messages have been updated in the design system.  [See guidance](https://design.va.gov/patterns/ask-users-for/phone-numbers#content-considerations)

I think the errorMessages should be: 

Required: Enter up to a 15-digit phone number (with or without dashes)
pattern: Enter a valid international phone number up to 15-digits (with our without dashes)

But this would be best verified by CAIA and confirmed with DST so it can be added to their guidance as well.